Common/cpp/Tools/JSLogger.h

```diff
@@ -0,0 +1,23 @@
+#ifdef DEBUG
+#pragma once
+
+#include "LayoutAnimationType.h"
+#include "Shareables.h"
+
+#include <memory>
+#include <string>
+
+namespace reanimated {
+
+class JSLogger {
+ public:
+  JSLogger();
```

Comment:
Do we need this constructor?

Reply:
No we don't, it's a leftover from when I actually was doing something in this constructor.